bridge-token-factory/tests/token_transfer.rs

```diff
@@ -237,6 +243,53 @@ fn test_eth_token_transfer() {
     assert_eq!(token.get_total_supply(&mut runtime), "900");
 }
 
+#[test]
+fn test_with_invalid_proof() {
```

Comment:
Should this test panic?

Reply:
I already fix the test, it should not panic.